database/galactic-sovereign/migrations/6_create_planets.up.sql

```diff
@@ -3,22 +3,17 @@ CREATE TABLE planet (
   id uuid NOT NULL,
   player uuid NOT NULL,
   name text NOT NULL,
-  created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
-  updated_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  updated_at TIMESTAMP WITH TIME ZONE NOT NULL,
   PRIMARY KEY (id),
   FOREIGN KEY (player) REFERENCES player(id)
 );
 
-CREATE TRIGGER trigger_planet_updated_at
-  BEFORE UPDATE OR INSERT ON planet
-  FOR EACH ROW
-  EXECUTE FUNCTION update_updated_at();
-
 CREATE TABLE homeworld (
   player uuid NOT NULL,
   planet uuid NOT NULL,
   PRIMARY KEY (player, planet),
-  UNIQUE (player, planet),
+  UNIQUE (player),
```

Comment:
This was discovered while testing: the homeworld should be unique by player. Most likely this comes from a bad copy paste from another table.